src/CreateEntityReducer.js

```diff
@@ -58,6 +58,15 @@ export function createEntityReducer(config) {
 
         var [, actionTypePrefix] = resultKey.toString().match(/(.*)_(FETCH|ERROR|RECEIVE)$/) || [];
 
+
+        if(type === 'ENTITY_DELETE') {
+            if(Iterable.isKeyed(state.getIn(payload))) {
+                return state.setIn(payload.concat('_deleted'), true);
```

Comment:
I guess I'm a bit confused about how the user consumes deleted entities. There doesn't seem to be anything in this update to filter out entities with the _deleted flag so is it up to the user to notice it and filter them?